certified-connectors/jbhunt/apiDefinition.swagger.json

```diff
@@ -10,7 +10,7 @@
       "email": "iomdri@microsoft.com"
     }
   },
-  "host": "api.jbhunt.com",
+  "host": "api.nonprod.jbhunt.com",
```

Comment:
@vechall We do not certify connectors that have test or staging or non-production host URLs. Either you need to have a dynamic host policy template to accept host URL from the users or have a production host URL.